src/ethtool/Makefile

```diff
@@ -11,6 +11,10 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	git clone https://salsa.debian.org/kernel-team/ethtool/
 	pushd ethtool
 	git checkout tags/debian/1%$(ETHTOOL_VERSION_BASE)-1
+ifneq (,(filter $(CONFIGURED_ARCH),arm64 armhf))
+	#Netlink based user-space interface support for ethtool is disabled will be using ioctl interface instead
+	sed -i 's/enable_netlink=yes/enable_netlink=no/g' configure.ac  
```

Comment:
>sed -i 's/enable_netlink=yes/enable_netlink=no/g' configure.ac [](start = 1, length = 62)

If you may better change the command in debian/rules
```
override_dh_auto_configure:
 ./autogen.sh
 dh_auto_configure -- --sbindir=/sbin
```

Reply:
why disable this on arm platform?

Reply:
@qiluo-msft 

Updated !